test/metadataSpec.js

```diff
@@ -26,6 +26,7 @@ describe('Metadata', function () {
           expect(res.body).to.deep.equal({
             currency_code: null,
             currency_symbol: null,
+            ilp_prefix: null,
```

Comment:
due to some weirdness in the config class, I'm blocked by https://github.com/interledger/five-bells-ledger/pull/310 to include the test I wrote